pgrx/src/array/port.rs

```diff
@@ -126,3 +126,22 @@ pub(super) unsafe fn ARR_DATA_PTR(a: *mut pg_sys::ArrayType) -> *mut u8 {
 
     unsafe { a.cast::<u8>().add(ARR_DATA_OFFSET(a)) }
 }
+
+/// Returns a pointer to the lower bounds of the array.
```

Comment:
...Maybe this should specify whether it actually points to the _list_ of lower bounds given by Postgres as a series of integers, or the actual lower bound position in the actual data.

( The answer is the former. )